packages/app-bridge/src/react/useBlockAssets.ts

```diff
@@ -53,15 +53,7 @@ export const useBlockAssets = (appBridge: AppBridgeBlock) => {
     };
 
     const updateAssetIdsFromKey = async (key: string, newAssetIds: number[]) => {
-        const currentBlockAssets = await appBridge.getBlockAssets();
-        const oldAssetIds = currentBlockAssets[key]?.map((asset) => asset.id) ?? [];
-
-        try {
-            await appBridge.deleteAssetIdsFromBlockAssetKey(key, oldAssetIds);
-            await appBridge.addAssetIdsToBlockAssetKey(key, newAssetIds);
-        } catch (error) {
-            console.error(error);
-        }
+        appBridge.api({ name: 'setAssetIdsByBlockAssetKey', payload: { key, assetIds: newAssetIds } });
```

Comment:
Do you think it makes sense to await this call, so that we can potentially chain operations after the request is completed?

Reply:
Thanks, guyss